src/views/Home/Teams/TeamList.jsx

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable no-restricted-globals */
+import { useEffect, useState } from 'react';
+import { getTeams, deleteTeamById } from '../../../services/teams';
+import { Link } from 'react-router-dom';
+import { useHistory } from 'react-router-dom';
+
+function TeamList() {
+  const [teams, setTeams] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const history = useHistory();
+
+  const loadTeams = async () => {
+    setLoading(true);
+    const resp = await getTeams();
+    setTeams(resp);
+    setLoading(false);
+  };
+
+  useEffect(() => {
+    loadTeams();
+  }, []);
+
+  const handleUpdate = (id) => {
+    history.push(`/teams/updateTeam/${id}`);
+  };
+
+  const handleDelete = async ({ id, name }) => {
+    const shouldDeleteTeam = confirm(
+      `Are you sure you want to delete ${name}?`
+    );
+
+    if (shouldDeleteTeam) {
+      await deleteTeamById(id);
```

Comment:
This does not currently work when there are existing players on the team and there isn't any user-facing error handling to indicate an issue with the request. 

You'd want to either 1: Prompt the user that there are existing players and ask them to remove the players first; or 2: Run the `deletePlayerById()` method here prior to deleting the team as a whole.